### DISCUSSION TITLE
Unit testing?

### DISCUSSION THREAD

**Discussion by hbroer** [nanojsx/nano#141] [Help / Questions]
Hey,

I could not find any information about how to unit test NanoJSX components. Does nobody test it?

Greetings
Hauke

**Comment by yandeu** [comment]
You will find some tests here:
https://github.com/nanojsx/nano/tree/master/test

**Reply by hbroer** [reply]
Thanks. But there is no hint about how to do unit tests in an application. Something like shallow rendering is not providet. It would only cover integration tests because it always renders deep. Or did I miss something?
